src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+}
+
+* {
+  /* margin: 0; */
+  /* padding: 0; */
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+input {
+  font-family: inherit;
+  font-weight: inherit;
+  width: 100%;
+}
+
+.big_form {
+  position: relative;
+}
+
+.bigform__input {
+  display: block;
+  margin: 20px 0;
+  padding: 23px 0 23px 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-size: 16px;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.bigform__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigform__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 900;
+  outline-style: none;
+  box-shadow: none;
+}
+
+.icon-search-big {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  height: 19px;
+  z-index: 1;
+}
+
+.small_form {
+  position: relative;
+}
+
+.smallform__input {
+  display: block;
+  padding: 11px 0 10px 33px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-size: 14px;
+  background-color: #fff;
+  margin-bottom: 20px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Comment:
Do not repeat your self
![2023-01-19_16h42_09](https://user-images.githubusercontent.com/61904995/213471666-bd72c273-2d9a-4dad-b564-95f36d0b869f.png)